### ISSUE TITLE
Smiley window is not scrollable.

### ISSUE THREAD

**Issue by ghost** [friendica/friendica#10031]
If you have activated the smiley addon in the admin settings and then want to insert a smiley in a post, you will notice that the popup window of the smiley cannot be scrolled.

**Comment by MrPetovan** [comment]
What theme are you using?

**Comment by ghost** [comment]
> What theme are you using?

Frio

**Comment by MrPetovan** [comment]
Yeah, the smiley overlay is wonky on frio when there are too many of them (like with the unicode emojis)
